locksmith/src/logger.ts

```diff
@@ -13,7 +16,10 @@ export const logger = winston.createLogger({
 // No output in tests
 logger.add(
   new winston.transports.Console({
-    silent: process.env?.NODE_ENV === 'test',
+    silent: !!(
+      process.env?.NODE_ENV &&
+      ['test', 'production'].indexOf(process.env?.NODE_ENV) > -1
```

Comment:
not on staging?

Reply:
I am not sure I want to murky the production logs... but maybe we can add later?